src/components/FeatureViews/Courses/CourseListOptions.js

```diff
@@ -0,0 +1,253 @@
+import moment from "moment";
+import gql from "graphql-tag";
+import { useQuery } from "@apollo/react-hooks";
+import Loading from "../../OmouComponents/Loading";
+import AccessControlComponent from "../../OmouComponents/AccessControlComponent"
+import { fullName, USER_TYPES, sortTime } from "../../../utils";
+import theme from "../../../theme/muiTheme";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    width: "100%",
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+  expanded: {
+    "&$expanded": {
+      margin: "0px 0",
+    },
+  },
+  anchorStyle: {
+    marginLeft: ".25em",
+    "&:hover": {
+      textDecoration: "underline",
+    },
+    "&:focus": {
+      color: "red",
+    },
+  },
+  buttons: {
+    display: "flex",
+    justifyContent: "flex-end",
+  },
+  accordionDivider: {
+    borderBottom: "1px solid #C4C4C4",
+  },
+  newNoteButton: {
+    marginBottom: "2em",
+    border: "1px solid #999999",
+    borderRadius: "5px",
+    fontSize: ".75rem",
+    fontWeight: 300,
+    fontFamily: "Roboto",
+    height: "2.5em",
+    marginTop: "2em",
+  },
+  plusSpan: {
+    fontSize: "1rem",
+    fontWeight: 500,
+    color: "#666666",
+    paddingRight: ".25em",
+  },
+}));
+
+export const GET_SESSION_NOTES = gql`
+  query getSessionNotes($sessionId: ID!) {
+    __typename
+    sessionNotes(sessionId: $sessionId) {
+      id
+      body
+      subject
+      poster {
+        firstName
+        lastName
+        id
+      }
+      createdAt
+      updatedAt
+    }
+  }
+`;
+
+const CourseListOptions = ({
+  sessionId,
+  loggedInUser,
+}) => {
+  const classes = useStyles();
+  const [readMore, setReadMore] = useState(false);
+  const [expand, setExpand] = useState(false);
+  const [noteId, setNoteId] = useState();
+  const [open, setOpen] = useState(false);
+  const [noteBody, setNoteBody] = useState("");
+  const [noteSubject, setNoteSubject] = useState("");
+  const [buttonState, setButtonState] = useState("");
+
+  const { data, loading, error } = useQuery(GET_SESSION_NOTES, {
+    variables: { sessionId: sessionId, },
+  });
+
+  if (loading) return <Loading />;
+  if (error) return console.error(error.message);
+
+  const handleOpenForm = (e) => {
+    e.preventDefault();
+    setOpen(true);
+  };
+
+  const handleCloseForm = (boolean) => setOpen(boolean);
+
+  const handleEdit = (e) => {
+    e.preventDefault();
+    const currentSubject = e.currentTarget.dataset.subject;
+    const currentBody = e.currentTarget.dataset.body;
+    const currentId = e.currentTarget.dataset.id;
+    setNoteBody(currentBody);
+    setNoteSubject(currentSubject);
+    setNoteId(currentId);
+    setOpen(true);
+    setButtonState("edit");
+  };
+
+  const handleReadMoreClick = () => setReadMore(!readMore);
+
+  const sessionNotesRender = data.sessionNotes.sort((firstVal, secondVal) => sortTime(firstVal.updatedAt, secondVal.updatedAt))
```

Comment:
same comment - use new lines when a single line is too long. improves readability